src/zkevm_specs/evm/execution/block_coinbase.py

```diff
@@ -0,0 +1,17 @@
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+
+
+def coinbase(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    instruction.constrain_equal(opcode, Opcode.COINBASE)
+    address = instruction.stack_push()
+    # in real circuit check address raw data is 160 bit length (20 bytes)
```

Comment:
btw, why check in real circuit but not here?

Reply:
in python, U160 type already stands for 20 bytes, for now evm circuit also have type `Address` represents it limitation